docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Atmospheric.Bmp085.md

```diff
@@ -15,7 +15,7 @@ The **BMP085** is a high-precision, low-power barometric pressure sensor. The BM
 ### Code Example
 
 ```csharp
-Bmp085 sensor;
+Bmp085? sensor;
```

Comment:
Are these various sensor classes actually structs under the hood, or is this `?` stuff just around the system for explicitly stating when things could possibly be null?

Reply:
yeah it's really just for null warning cleanup